src/h5web/visualizations/containers/NxSpectrumContainer.tsx

```diff
@@ -21,10 +21,6 @@ function NxSpectrumContainer(props: VisContainerProps): ReactElement {
     disableErrors(!errors);
   }, [disableErrors, errors]);
 
-  if (!signal.value) {
-    return <></>;
-  }
-
   if (errors && signal.value.length !== errors.length) {
```

Comment:
I'm able to remove the early return by marking `value` as being required in `SignalParams`. This is fine because we throw if we encounter an undefined signal value in `useNxData`.